README.md

```diff
@@ -36,6 +36,7 @@ Project Name | Description | Languages | Frameworks | Supported Platforms
 
 Project Name | Description | Languages | Frameworks | Supported Platforms
 --- | --- | :-: | :-: | :-:
+[OpenLP](https://gitlab.com/openlp) | Worship presentation software designed to fit how you want to run your service. | Python | - | [Desktop & Mobile (Remote)](https://openlp.org/)
```

Comment:
"Remote" is unclear here - after looking at the link, I assume this means that the mobile platform is for the remote control?

Reply:
Yes, i believe so - like a 'clicker' that a presenter would use.